src/app/_components/BlockList/utils.ts

```diff
@@ -57,32 +54,33 @@ export function waitForFadeAnimation(callback: () => void) {
   setTimeout(callback, FADE_DURATION);
 }
 
-export function generateBlockList(
-  stxBlocks: (Block | NakamotoBlock)[],
-  btcBlocksMap?: BtcBlockMap
-) {
-  if (stxBlocks.length === 0) return [];
+export function generateBlockList(stxBlocks: (Block | NakamotoBlock)[], btcBlocksMap: BtcBlockMap) {
```

Comment:
Would you mind adding a unit test to this function?

Reply:
sure